app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="match_constraint">0dp</dimen>
+
+    <dimen name="margin_0dp">0dp</dimen>
+    <dimen name="margin_8dp">8dp</dimen>
+    <dimen name="margin_16dp">16dp</dimen>
+
+    <dimen name="news_picture_height">160dp</dimen>
+    <dimen name="category_height">60dp</dimen>
+    <dimen name="buttons_height">40dp</dimen>
+
+    <dimen name="text_size_title">24dp</dimen>
```

Comment:
Please, revisit the dimens guide and pick up correct unit for text/font sizes. Texts and fonts are not mesured in `dp`.